bridges/otelzap/encoder.go

```diff
@@ -18,47 +18,69 @@ var (
 	_ zapcore.ArrayEncoder  = (*arrayEncoder)(nil)
 )
 
+type newNameSpace struct {
+	ns   string
+	kv   []log.KeyValue
```

Comment:
Can we rename it to "attrs" (for attributes)?

```suggestion
	attrs []log.KeyValue
```